micrometer/README.adoc

```diff
@@ -0,0 +1,347 @@
+include::../shared-doc/attributes.adoc[]
+
+= micrometer: Micrometer QuickStart
+:author: Jason Lee
+:level: Beginner
+:technologies: Micrometer
+:custom-bootable-jar-layers: {lt}layer{gt}micrometer{lt}/layer{gt}
+
+[abstract]
+The `micrometer` quickstart demonstrates the use of the Micrometer library in {productName}.
+
+:standalone-server-type: default
+:archiveType: jar
+:archiveName: {artifactId}
+
+== What is it?
+
+https://micrometer.io[Micrometer] is a vendor-neutral facade that allows application developers to collect and report application and system metrics to the backend of their choice in an entirely portable manner. By simply replacing the `MeterRegistry` used, or combining them in Micrometer's `CompositeRegistry` data can be exported a variety of monitoring systems with no application code changes.
+
+== Architecture
+
+In this quickstart, we will build a small, simple application that shows the usage of a number of Micrometer's `Meter` implementations. We will also demonstrate the means by which {productName} exports the metrics data, which is via the https://opentelemetry.io/docs/reference/specification/protocol/otlp/[OpenTelemetry Protocol (OTLP)] to the https://opentelemetry.io/docs/collector/[OpenTelemetry Collector]. To provide simpler access to the published metrics, the Collector will be configured with a Prometheus endpoint, from which we can scrape data.
+
+== Prerequisites
+
+To complete this guide, you will need:
+
+* less than 15 minutes
+* JDK 11+ installed with `JAVA_HOME` configured appropriately
+* Apache Maven 3.5.3+
+
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+
+== Steps
+
+include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
+
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
+
+== Solution
+
+We recommend that you follow the instructions that <<creating-new-project, create the application step by step>>. However,
+you can also go right to the completed example which is available in this directory.
+
+// Build and Deploy the Quickstart
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
+
+[[creating-new-project]]
```

Comment:
Up to you but you don't really need to document how to create app from scratch, you may simply stick with only the includes + an access/interact with the app section between baremetal deploy and test includes. It's actually kind of redundant since documenting how to build app from scratch will be the task of a new App Development Guide in the works by DOCs team.

Reply:
OK. I think I followed the pattern I saw in another QS. If the code is the documentation, I may remove all of this to avoid the duplication.